lib/elixir/lib/module/parallel_checker.ex

```diff
@@ -10,8 +10,7 @@ defmodule Module.ParallelChecker do
   the modules and adds the ExCk chunk to the binaries. Returns the updated
   binaries and a list of warnings from the verification.
   """
-  @spec verify([{%{}, binary()}], [{module(), binary()}], pos_integer()) ::
-          {[{module(), binary()}], [warning()]}
+  @spec verify([{%{}, binary()}], [{module(), binary()}], pos_integer()) :: [warning()]
```

Comment:
```suggestion
  @spec verify([{map(), binary()}], [{module(), binary()}], pos_integer()) :: [warning()]
```

I think `%{}` matches the empty map only?